src/components/cards/InfoCard.js

```diff
@@ -37,77 +41,68 @@ const Image = styled.img`
   clip-path: circle(50% at 50% 50%);
 `
 
+const Gear = styled(gear)`
+  font-size: 30px;
+  position: absolute;
+  right: 15px;
+  top: 15px;
+`
+
 class InfoCard extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
       edit: false,
-      description: null
+      value: null
```

Comment:
I think it was from a previous solution I had.  I went ahead and removed the references to state.